Sources/SmithyIdentity/AWSCredentialIdentityResolver.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
This was placed in `SmithyIdentityAPI` previously, but it is a "concrete" implementation of `IdentityResolver` protocol, hence moved to `SmithyIdentity` module.